libs/ButeMgr/butemgr.h

```diff
@@ -669,7 +670,7 @@ inline bool CButeMgr::Parse(void* pData, unsigned long size, int decryptCode,
 	if (!pData)
 		return false;
 #if _MSC_VER >= 1300 && !defined(__clang__)
-	std::istrstream IStream{(char*)pData, size};
+	std::istrstream IStream{(char*)pData, (int)size};
```

Comment:
please use an unsigned type

Reply:
this really should move to a std::istringstream as strstream has been deprecated in C++98
https://en.cppreference.com/w/cpp/io/istrstream